### ISSUE TITLE
Use the new building source for the rest of LA county

### ISSUE THREAD

**Issue by maning** [osmlab/labuildings#111]
LA County will release a new building source layer from latest LARIAC.  We will use this for the rest of the unimported areas within the county.

**Next actions**

- [x] Get the data from LA County GIS unit. @jschleuss 
- [x] Match building data to parcel using AIN as reference attribute.  Should we also use the [new parcel data](https://github.com/osmlab/labuildings/issues/105)? @jschleuss 
- [x] Process data into OSM XML split by census blocks @maning 
- [x] Prepare TM projects. @maning 
- [x] Announce to the world.

@jschleuss @cityhubla @almccon

**Comment by maning** [comment]
New data is ready for processing: http://egis3.lacounty.gov/dataportal/2016/11/03/countywide-building-outlines-2014-update-public-domain-release/

@jschleuss @cityhubla

**Comment by maning** [comment]
@jschleuss bump.

**Comment by maning** [comment]
Data is now ready!  

Some notes:
* I modified the script to improve indexing during the chunking process. (https://github.com/osmlab/labuildings/pull/113)
* `chunk.py` fails if there are invalid geometries in the input. I deleted the invalid geometries (12 polygons) from the new building shapefile.
* To speed up the process, I only refreshed the `.osm` files for the blocks we have not yet imported (1869 in total).
* There were buildings that were outside the census block poly, these buildings were not included in the current refresh.

![screen shot 2016-12-10 at 13 27 12](https://cloud.githubusercontent.com/assets/353700/21073478/57b80782-bf05-11e6-87ab-c3accd21f7aa.png)

* I prepared a short diary to update the community about the recent refresh.  **Can someone review?** https://gist.github.com/maning/c27d0df57a362c486033530167f2b794
